### PR TITLE
Fix gallery buttons selector

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix https://github.com/nextcloud/gallery/issues/413
With https://github.com/nextcloud/gallery/pull/414

@nextcloud/designers

### CODE REVIEW

**Review by juliusknorr — APPROVED** [review]

File: core/css/styles.scss [hunk 1]
```diff
@@ -253,14 +253,16 @@ body {
 	box-sizing: border-box;
 }
 
-#controls {
+#controls .actions {
```
Comment (juliushaertl) [inline core/css/styles.scss]: .actions is also used by the files app "add" button and causes it to be not square anymore.
bildschirmfoto vom 2018-03-22 12-01-14

**Review by juliusknorr — DISMISSED** [review]

**Review by danxuliu — COMMENTED** [review]
Naive question, but would not removing `>` from [`> .button, button {`](https://github.com/nextcloud/server/blob/966ceca8eaf9d06d9f0528c83168a2942ca7e279/core/css/styles.scss#L259), without touching anything else in the server nor in the gallery, fix the problem? I mean, why are those rules applied only to direct children instead of descendants (like they were before 966ceca8eaf9d06d9f0528c83168a2942ca7e279)?

**Comment by skjnldsv** [comment]
@danxuliu the issue with the control bar is we have a lot of plugins registering their buttons here, and it's absolutely not standardised. So having a popovermenu here was conflicting with this rule, that's why I added the direct child selector to avoid the selector to go down the entire dom structure :)

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Pull request merged**